internal/repositories/order_test.go

```diff
@@ -44,16 +44,17 @@ func (s *OrderSuite) SetupTest() {
 	s.orderRepository = NewOrderRepository()
 	s.orderStatusRepository = NewOrderStatusRepository()
 
+	name := "user1"
 	s.user = &ent.User{
-		Login: "user1", Email: "user1@email.com", Password: "1234", Name: "user1",
+		Login: "user1", Email: "user1@email.com", Password: "1234", Name: &name,
```

Comment:
Why creating a separate variable instead of assigning value directly in User struct?